server/apps/nominations/controllers/user.js

```diff
@@ -13,7 +13,7 @@ import type { TableRequest } from '../../lib/tableApi';
 
 const RELATED_MODELS = [
   { model: db.affiliation, as: 'affiliation' },
-  { model: db.household, as: 'nomination' }
+  { model: db.household, as: 'nomination', where: {'deleted': false}, required: false }
```

Comment:
AH, so the `required` flag is what makes Sequelize _not_ omit the parent record if its relation is missing, huh? I was running in circles trying to figure that one out.

Reply:
http://docs.sequelizejs.com/manual/tutorial/models-usage.html#nested-eager-loading

Yep, totally missed that. Nice!